[Bug]: Husky error occurs when installing package via npm i into a project

### Before submitting...

- [X] I have searched for [duplicate or closed issues](https://github.com/materializecss/materialize/issues?utf8=%E2%9C%93&q=is%3Aissue).
- [X] I have read the [CONTRIBUTING document](https://github.com/materializecss/materialize/blob/master/CONTRIBUTING.md) and my issue is following the guidelines.
- [X] I have read the template completely before filling it in.

### Context

When installing 1.2.0 version of materializecss into our project via `npm i` an issue occurs.

This does not occur when installing version 1.1.0.

### Current Behavior

```
npm ERR! code 1
npm ERR! path [redacted]\node_modules\@materializecss\materialize
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c husky install
npm ERR! 'husky' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```


### Expected behavior

Expected the `npm i `  command to complete without an error. 

### Possible Solutions or Causes

Husky is included under devDependencies in package.json but is being ran as a postinstall script by the end user.

According to the Husky docs, you cannot include husky install as a postinstall without using pinst.  I am not sure if this is the root cause or not, but wanted to bring this to your attention:

https://typicode.github.io/husky/#/?id=yarn-2

![image](https://user-images.githubusercontent.com/54276042/204343145-9f4258ac-6b9e-4f65-9d2f-67a7c6ad908d.png)

I assume since husky is a dev dependency, you never intended for your end users to run the postinstall script. 

### Steps to reproduce

1. Create a project that references `@materializecss/materialize` in its package.json.
2. `npm i ` to install the packages
3. Note the error message. 

### Your Environment

* Version used: 1.2.0
* Browser Name and version: N/A
* Operating System and version (desktop or mobile): Windows 10 Desktop
* Additional information you want to tell us:

--- a/package.json
+++ b/package.json
@@ -22,7 +22,9 @@
     "precommit": "lint-staged",
     "docs": "grunt docs",
     "commit": "npx cz",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "prepack": "pinst --disable",
+    "postpack": "pinst --enable"
   },
   "lint-staged": {
     "js/*.js": [
@@ -65,6 +67,7 @@
     "lint-staged": "^7.0.5",
     "node-archiver": "^0.3.0",
     "phantomjs-prebuilt": "^2.1.14",
+    "pinst": "^3.0.0",
     "prettier": "^1.12.1",
     "sass": "^1.35.2"
   },

